draft-ietf-bidding-and-auction-services.md

```diff
@@ -391,10 +391,11 @@ the {{framing}} header.
 This section describes how the client MAY form and serialize request messages
 in order to communicate with the Bidding and Auction services.
 
-This algorithm takes as input all of the `relevant interest groups`, and a config
-consisting of the `publisher`,
+This algorithm takes as input all of the `relevant interest groups`, a config
+consisting of the `publisher`, a map of from (origin, string) tuple to origin
+`ig pagg coordinators`,
 an optional `desired total size`, an optional list of `interest group owners` to
-include each with an optional `desired size`, and the [HPKE] `public key` and

```

Comment:
The extra "and" was intentional here, as a way of grouping the key with its key ID. We could change the second "and" to "with" if you think that would be more clear.